coderedcms/project_template/project_name/settings/base.py

```diff
@@ -138,15 +138,17 @@
     },
 ]
 
-
 # Internationalization
 # https://docs.djangoproject.com/en/{{ docs_version }}/topics/i18n/
 
-LANGUAGE_CODE = 'en-us'
+# To add or change language of the project, modify the list below.
+LANGUAGES = [
+    ('en', _('English'))
+]
```

Comment:
After reading the Django docs, I'm thinking we should leave `LANGUAGE_CODE` as-is. It seems like that is the fallback when Django can't figure out what language to use.